src/WebDriver/WebDriverExpectedCondition.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Panther project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Panther\WebDriver;
+
+use Facebook\WebDriver\Exception\StaleElementReferenceException;
+use Facebook\WebDriver\WebDriver;
+use Facebook\WebDriver\WebDriverBy;
+use Facebook\WebDriver\WebDriverExpectedCondition as BaseWebDriverExpectedCondition;
+
+final class WebDriverExpectedCondition extends BaseWebDriverExpectedCondition
```

Comment:
Extending `BaseWebDriverExpectedCondition` isn't necessary, isn't it? Using composition is preferable.